### PR TITLE
aboutUs update

### PR DESCRIPTION
**Pull request by Amrutha1101**
Code needs refactoring (a lot actually)

### CODE REVIEW

File: components/AboutInfo.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { withRouter } from "next/router";
+
+const AboutInfo = ({ router }) => {
```
Comment (arpancodes) [inline components/AboutInfo.js:4]: why do we need router here? 🤔

File: components/AboutInfo.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                    </p>
+                </div>
+            </div>   
+        </div>
+    </div>
+    
+  );
+};
+
+export default withRouter(AboutInfo);
```
Comment (arpancodes) [inline components/AboutInfo.js:82]: you can remove the withRouter hoc here

File: components/AboutInfo.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { withRouter } from "next/router";
```
Comment (arpancodes) [inline components/AboutInfo.js:2]: no need for this import.

File: components/AboutInfo.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        <div className="lg:w-4/5 w-11/12  bg-gray-900 rounded-lg px-4  py-2  my-10 m-auto text-center">
+            <h1 className="text-5xl text-yellow-500">Our Mission</h1>
+            <div className="lg:flex justify-between items-start">
+                <div className="md:w-96 md:w-1/2 lg:mr-2 my-5 m-auto  bg-purple-600 bg-opacity-25 rounded-lg">
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_edu.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75  rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-green-600 text-black text-opacity-100 uppercase">Education</h3>
+                        <p className="mx-5 pb-5">
+                        GDSC CMRIT provides a platform for the growth and advancement of the student community in the field of technology by providing peer-to-peer learning environment by conducting , webinars and events , providing hands on experiences on topics like Web Development, Mobile App Development, Data Science, Blockchain,  Machine Learning, Contributing to Open Source and much more!</p>
+                    </div>
```
Comment (arpancodes) [inline components/AboutInfo.js:29]: This is very similar piece of code just differing in text, so you can create a separate component and just pass title, image and text as prop.
we use components to reduce code duplication.

File: components/AboutInfo.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                <div className="md:w-96 md:w-1/2 lg:mr-2 my-5 m-auto bg-purple-600 bg-opacity-25 rounded-lg">
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_collab.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75 rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-green-600 text-black text-opacity-100 uppercase"> Collaboration</h3>
+                        <p className="mx-5 pb-5 ">
+                        We are not just a community but a mini India where people from diverse backgrounds and different majors come together to collaborate and co-create new innovative ideas that would help solve real world problems, build solutions for the local problems and beget a community that aspires for a positive and impactful change.</p>
+                    </div>
+                    
+
+                </div>
```
Comment (arpancodes) [inline components/AboutInfo.js:41]: Same with this.

File: components/AboutInfo.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+                    <img className="w-52 h-40 mx-auto my-8" src="/mission_innovate.png"></img>
+                    <div className="bg-purple-600 bg-opacity-75 rounded-b-lg">
+                        <h3 className="text-3xl py-3.5 text-black text-opacity-100 uppercase"> Innovations</h3>
+                        <p className=" mx-5 pb-5">
+                        Through education and collaboration we want to provide students a platform where they can innovate and experiment along with us. Developing innovative products is what separates one from the crowd. Therefore, we want to lay the foundation necessary to bring our innovative visions to life through the power of technology.</p>
+                    </div>
+                    
+                </div>
+            </div>
+        </div>
```
Comment (arpancodes) [inline components/AboutInfo.js:52]: and again same with this. all these three can be taken out into separate components.

File: components/AboutInfo.js [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+            <div className="md:w-2/5 my-4 m-auto py-3 px-5 bg-gray-900 text-center rounded-lg shadow-lg">
+                <h1 className="text-5xl text-yellow-500">Our Vision</h1>
+                <div className="">
+                    <img src="/vision.jpg"  className="md:w-80 h-80  py-3 m-auto"></img>
+                    <p className="text-xl my-4 text-center ">
+                    The DSC program is focused with a vision to inculcate individual’s with skills and knowledge, who are intrigued about the possibilities that developer technologies can bring changes to this world. We strive for the betterment of society through technology.
+
+                    </p>
+                </div>
+            </div>
```
Comment (arpancodes) [inline components/AboutInfo.js:75]: Similar pattern for these two divs... extract them out into components.

**Review by arpancodes — CHANGES_REQUESTED** [review]
I'm glad that you are getting a hang of React.
But let's start creating components ad much as possible.
Whenever you find repeating yourself... create a component!

File: components/Mission.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (arpancodes) [inline components/Mission.js:27]: remove the blank lines?

File: components/Mission.js [hunk 9]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+const Mission = ({MissionImage, MissionTitle,MissionText}) => {
```
Comment (arpancodes) [inline components/Mission.js:2]: change the variable name to camelCase with starting with `lowercase` convention is to use classes and component names starting from uppwercase.

**Review by arpancodes — CHANGES_REQUESTED** [review]
rest looks good to me

File: components/AboutInfo.js [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+        </div>
+        {/* Mission ends */}
+        {/* Aim and vision Starts */}
+
+        <div className="md:w-4/5  w-11/12  md:flex items-start m-auto ">
+            <VisionAim
+                title = "Our Aim"
+                image = "/aim.jpg"
+                text = "We aim to provide students with the opportunity to: Grow their knowledge on developer technologies and more through peer to peer workshops and events, gain relevant industry experience by solving problems for local organizations with technology based solutions and showcase their prototypes and solutions to their local community and industry leaders."
```
Comment (amartyanambiar) [inline components/AboutInfo.js:46]: Maybe reduce the text content to make all the components look even .
Same goes for the 'Education' section

**Review by amartyanambiar — CHANGES_REQUESTED** [review]

File: components/AboutInfo.js (hunk 10 above)
Comment (arpancodes) [reply components/AboutInfo.js:46]: yaa or make the flex to stretch

File: components/Mission.js (hunk 9 above)
Comment (Amrutha1101) [reply components/Mission.js:2]: Forgot that😅

**Review by amartyanambiar — CHANGES_REQUESTED** [review]
All font sizes shall be same in about,vision,mission,aim.
And coming to hover colors , lets follow one scheme only.( presently yellow,blue)

**Pull request merged**